Automattic/wp-calypso
# A4A > Marketplace: Hosting page UI enhancements

Closes https://github.com/Automattic/jetpack-genesis/issues/464
Closes https://github.com/Automattic/jetpack-genesis/issues/460

## Proposed Changes

This PR:

- Makes some text changes to match the design
- Does some UI enhancements to fix some design issues and match the design.
- Does some fixes to the mobile view by making the font size smaller.

NOTE: 

- Changes to Refer Mode will be done in another PR
- Final Testimonials will be updated in another PR

## Why are these changes being made?

* These changes are being made to match the hosting page design.

## Testing Instructions

* Open the A4A live link
* Verify the UI looks good on all the screen sizes and matches the design: 3JELpER0gWwwI1BMrTyGE0-fi-6836_71380

![screencapture-agencies-localhost-3000-marketplace-hosting-2024-08-02-09_35_57](https://github.com/user-attachments/assets/f527e65d-b047-46e3-86d9-81a46c10a9b7)
![screencapture-agencies-localhost-3000-marketplace-hosting-2024-08-02-09_36_21](https://github.com/user-attachments/assets/1d786c98-0199-4a32-93c0-3f9c02a04c5a)
![screencapture-agencies-localhost-3000-marketplace-hosting-2024-08-01-12_09_34](https://github.com/user-attachments/assets/f4f08f39-9d7a-4633-93be-56c6723505b8)


![iPhone-12-PRO-agencies localhost (2)](https://github.com/user-attachments/assets/535a59b0-3ee9-4b89-b61f-25334bbec249)
![iPhone-12-PRO-agencies localhost (4)](https://github.com/user-attachments/assets/8afc999b-fadb-4a74-9854-1de9a6e541af)


## Pre-merge Checklist
